flocker/volume/filesystems/zfs.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import absolute_import
 
 import os
+import libzfs_core
```

Comment:
I guess we'll need to think about how to specify this dependency.
Simplest thing would be to add it to `requirements.txt` but first we'll have to make sure that `libzfs_core` is available from PyPI.
Perhaps it's better to have some separate `requirements.txt` files for each of the Flocker dataset backend plugins.